README.md

```diff
@@ -1,6 +1,6 @@
 # Monero
 
-Copyright (c) 2014-2022 The Monero Project.   
+Copyright (c) 2014-2022 The Monero Project.
```

Comment:
The double spaces are needed here to produce a newline. Keep them.

Reply:
Ah, I was wondering what they were for in the first place, thanks.